python/nav/mibs/ALCATEL-IND1-PORT-MIB.py

```diff
@@ -0,0 +1,75 @@
+"""A class for getting DDM values for ALE equipment
```

Comment:
Actually, I forgot one nitpick:

Please see: https://nav.uninett.no/doc/latest/hacking/hacking.html#python-boilerplate-headers :-)

Reply:
I hope it's all there now. :-)

Reply:
It is, thanks!